addons/nodot/interaction/Teleport3D.gd

```diff
@@ -0,0 +1,88 @@
+## Teleport3D node
+class_name Teleport3D extends Area3D
+
+@export_category("Nodes")
+## Target node for copy the coords
+@export_node_path("Node3D") var Target
+## Player node or other objective to teleport
+@export_node_path("CharacterBody3D") var Player
+
+@export_category("Others Variables")
+## Size of collision area
+@export var size = Vector3(3.372, 3.821, 1)
+## Time to teleport (in seconds, and this variable is optional)
+@export var time_teleport : int = 0
+
+## Get position of "Player" 
```

Comment:
The var name describes a position, but actually this is just a reference to the player node.
Additionally, this should be typed to be `CharacterBody3D`.